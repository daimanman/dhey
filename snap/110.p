{"C":1,"Duration":114695832,"EndTime":1551408760070031000,"Err":"","FileId":"","InputFileName":"","LonhApi":"signOnline","Method":"","N":1,"P":null,"Payload":"","QH":{},"Remark":"","ReqNums":1,"StartTime":1551408759955335200,"Status":1,"TaskId":110,"Type":"LONH","Url":"","Z":""}