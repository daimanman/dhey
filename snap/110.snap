{"AvgTotal":0.1057764,"Fastest":0.1057764,"Slowest":0.1057764,"Average":0.1057764,"Rps":9.453904651699244,"AvgConn":0.0518614,"AvgDNS":0.004985,"AvgReq":0,"AvgRes":0,"AvgDelay":0.0529183,"ConnMax":0.004985,"ConnMin":0.004985,"DnsMax":0.004985,"DnsMin":0.004985,"ReqMin":0,"ReqMax":0,"ResMin":0,"ResMax":0,"DelayMax":0.0529183,"DelayMin":0.0529183,"Lats":[0.1057764],"ConnLats":[0.004985],"DnsLats":[0.004985],"ReqLats":[0],"ResLats":[0],"DelayLats":[0.0529183],"Total":105776400,"ErrorDist":{},"StatusCodeDist":{"200":1},"SizeTotal":0,"SizeReq":0,"NumRes":1,"LatencyDistribution":[{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0}],"Histogram":[{"Mark":0.1057764,"Count":1,"Frequentcy":1},{"Mark":0.1057764,"Count":0,"Frequentcy":0},{"Mark":0.1057764,"Count":0,"Frequentcy":0},{"Mark":0.1057764,"Count":0,"Frequentcy":0},{"Mark":0.1057764,"Count":0,"Frequentcy":0},{"Mark":0.1057764,"Count":0,"Frequentcy":0},{"Mark":0.1057764,"Count":0,"Frequentcy":0},{"Mark":0.1057764,"Count":0,"Frequentcy":0},{"Mark":0.1057764,"Count":0,"Frequentcy":0},{"Mark":0.1057764,"Count":0,"Frequentcy":0},{"Mark":0.1057764,"Count":0,"Frequentcy":0}],"TaskId":110,"Duration":0.1057764}